packages/renderer/src/stores/kubernetes-contexts-state.ts

```diff
@@ -166,6 +166,31 @@ export const kubernetesCurrentContextNodesFiltered = derived(
   ([$searchPattern, $nodes]) => $nodes.filter(node => findMatchInLeaves(node, $searchPattern.toLowerCase())),
 );
 
+// Pods
+
+export const kubernetesCurrentContextPods = readable<KubernetesObject[]>([], set => {
+  window
+    .kubernetesRegisterGetCurrentContextResources('pods')
+    .then(value => set(value))
+    .catch((err: unknown) => console.log('Error registering Kubernetes pods', err));
```

Comment:
pure question: do we expect it's only a à.log` or it's an `.error`

Reply:
This pattern was merged 5 days ago in #9700 for all the Kubernetes stores. I agree it probably is unexpected enough to be an `.error`, but separate PR.